lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -169,7 +169,11 @@ def request_json_matches(required_attributes)
       #)
       def publishing_api_has_content(items, params = {})
         url = PUBLISHING_API_V2_ENDPOINT + "/content"
-        stub_request(:get, url).with(:query => params).to_return(status: 200, body: { results: items }.to_json, headers: {})
+        body = { results: items, total: 1, pages: 1, current_page: 1 }
```

Comment:
We could do that, yes. Also, since we can send a specific `page` param, I can check for it and change `current_page` accordingly. Does that make sense?